package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "@lumigo/node-core": "1.13.0",
     "agentkeepalive": "^4.1.4",
-    "axios": "0.24.0",
+    "axios": "1.6.5",
```

Comment:
Are we sure we want to force customers to update to this version? it might conflict with their existing axios version.

We could set it to a base version or above, but then we have to test for every version to make sure we don't break...

Reply:
It does not mean that client must use this version - in Node, each package can have its own version of a dependency, and is isolated from the others (contrary to Python)

Reply:
Got it, cool